.codeclimate.yml

```diff
@@ -0,0 +1,46 @@
+version: "2"
+exclude_patterns:
+- "doc/"
+- "dist/"
+- "**/tests/"
```

Comment:
Should we include tests in CodeClimate? Tests are code too.

Reply:
Yeah I agree.